app/views/pages/release-notes/29-JULY-2019.liquid

```diff
@@ -12,7 +12,20 @@ searchable: false
 
 <h4 class="release-note release-note__new">NEW</h4>
 
-* todo
+* **graphql `models()` can be filtered by related users**: for example you can filter `cars` by owner name
+  ```graphql
+    models(
+      per_page: 10
+      filter: {
+        name: { value: "cars" }
+        users: { join: { foreign_property: "car_id" }
+          filter: { properties: [{ name: "name" value: "Mike" }]
+          }
+        }
+      }
+    ) { ... }
+  ```
+* **multilingual notifications** - added `locale` attribute to Email/SMS/API Notifiations, which accepts liquid, to evaluate to the locale which should be used for translations
```

Comment:
Example would be nice

Also probably docs will be out of date because of this (no mention of locale).

BTW. You set *locale*, for our system to get *translations* with proper *language*.

Reply:
Updated documentation in #671